feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/sendrequest/SendRequestFragment.java

```diff
@@ -143,19 +143,19 @@ public void onCallback() {
      */
 
 
-    private void storeRecipientUsername(String friendName){
-        //TODO: pass filename variable from MainActivity
-        SharedPreferences mySharedPreferences = this.getActivity().getSharedPreferences("Xiaole", Context.MODE_PRIVATE);
-        SharedPreferences.Editor myEditor = mySharedPreferences.edit();
-        myEditor.putString("recipientUsername", friendName);
-        myEditor.apply();
-    }
-
-    private String getRecipientUsername(){
-        SharedPreferences mySharedPrefrences = this.getActivity().getSharedPreferences("Xiaole", Context.MODE_PRIVATE);
-        //String RecipientUsername = mySharedPrefrences.getString("recipientUsername", this.getActivity().);
-        String RecipientUsername = "Xiaole2";
-        return RecipientUsername;
-    }
```

Comment:
I'm assuming this is separate from the problem I told you about? because the get() DAO method for followpermissions is still returning a users followers instead of followees

Reply:
The changes are made in the followRequest fragment logic, not the DAO logic

Reply:
do you know what the changes need to be done are though?

Reply:
I have the changes made in RequestList.java